src/utilities/getVTokenByAddress.ts

```diff
@@ -2,8 +2,8 @@ import { VToken } from 'types';
 
 import { VBEP_TOKENS } from 'constants/tokens';
 
-const getVTokenByAddress = (address: string) =>
-  address.toLowerCase() in VBEP_TOKENS
+const getVTokenByAddress = (address: string | null) =>
+  !!address && address.toLowerCase() in VBEP_TOKENS
     ? (VBEP_TOKENS[address.toLowerCase() as keyof typeof VBEP_TOKENS] as VToken)
     : undefined;
```

Comment:
How are you getting vBnb if there is no underlying address for it?

Reply:
This function won't receive an underlying address, it receives the address for vBNB, which is a valid key in the VBEP_TOKENS object. `getTokenByAddress` is the one that needs the fallback, since it deals with underlying tokens

Reply:
If the address is null it returns undefined but you it should return the token for vBnb